rspace/src/test/scala/coop/rchain/rspace/HotStoreSpec.scala

```diff
@@ -132,27 +118,15 @@ trait HotStoreSpec[F[_], M[_]] extends FlatSpec with Matchers with GeneratorDriv
         cachedContinuations: Vector[Continuation],
         installedContinuation: Continuation
     ) =>
-      fixture { (state, history, hotStore) =>
+      fixture { (state, _, hotStore) =>
         {
           for {
-            _ <- state.modify(
-                  _ =>
-                    (
-                      HotStoreState(
-                        continuations = Map(
-                          channels -> cachedContinuations
-                        ),
-                        installedContinuations = Map.empty,
-                        data = Map.empty,
-                        joins = Map.empty,
-                        installedJoins = Map.empty
-                      ),
-                      ()
-                    )
-                )
-            _     <- hotStore.installContinuation(channels, installedContinuation)
-            res   <- hotStore.getContinuations(channels)
-            cache <- state.get
+            _ <- state.modify { _ =>
+                  (HotStoreState(continuations = Map(channels -> cachedContinuations)), ())
+                }
```

Comment:
If previous state is not used when modified then it's discarded, so `set` method on Ref can be used.

Reply:
Done

Reply:
@tgrospic, is empty parentheses, in this case, is the previous state? I thought it is what should `modify` returns in an atomic way.